Header header
uint8 num_of_units
uint8 address
uint8 remote_address
UWBRangeWithOffset[] ranges
